src/axom/inlet/Table.cpp

```diff
@@ -74,15 +99,24 @@ std::shared_ptr<Field> Table::addBool(const std::string& name,
     sidreGroup->createViewScalar("value", value? int8(1) : int8(0) );
   }
 
-  return std::make_shared<Field>(sidreGroup, m_sidreRootGroup,
+  auto field =  std::make_shared<Field>(sidreGroup, m_sidreRootGroup,
                                  axom::sidre::DataTypeId::INT8_ID, m_docEnabled);
+  currTable->m_fieldChildren[fullName] = field;
+  return field;                      
 }
 
 std::shared_ptr<Field> Table::addDouble(const std::string& name,
                                         const std::string& description)
 {
   std::string fullName = getFullName(m_name, name);
-  axom::sidre::Group* sidreGroup = baseFieldAdd(fullName, description);
+  size_t found =  name.find_last_of("/");
+
+  auto currTable = shared_from_this();
```

Comment:
Some of this can be moved into baseFieldAdd because its common logic between all add Field functions.